cpp/src/arrow/compute/kernels/aggregate_test.cc

```diff
@@ -383,18 +404,54 @@ class TestFloatingMinMaxKernel : public TestNumericMinMaxKernel<ArrowType> {};
 TYPED_TEST_CASE(TestNumericMinMaxKernel, IntegralArrowTypes);
 TYPED_TEST(TestNumericMinMaxKernel, Basics) {
   MinMaxOptions options;
+  std::vector<std::string> chunked_input1 = {"[5, 1, 2, 3, 4]", "[9, 1, null, 3, 4]"};
+  std::vector<std::string> chunked_input2 = {"[5, null, 2, 3, 4]", "[9, 1, 2, 3, 4]"};
+  std::vector<std::string> chunked_input3 = {"[5, 1, 2, 3, null]", "[9, 1, null, 3, 4]"};
+
   this->AssertMinMaxIs("[5, 1, 2, 3, 4]", 1, 5, options);
   this->AssertMinMaxIs("[5, null, 2, 3, 4]", 2, 5, options);
+  this->AssertMinMaxIs(chunked_input1, 1, 9, options);
+  this->AssertMinMaxIs(chunked_input2, 1, 9, options);
+  this->AssertMinMaxIs(chunked_input3, 1, 9, options);
+
+  options = MinMaxOptions(MinMaxOptions::OUTPUT_NULL);
+  this->AssertMinMaxIs("[5, 1, 2, 3, 4]", 1, 5, options);
+  // output null
+  this->AssertMinMaxIs("[5, null, 2, 3, 4]", 0, 0, options);
+  // output null
+  this->AssertMinMaxIs(chunked_input1, 0, 0, options);
+  this->AssertMinMaxIs(chunked_input2, 0, 0, options);
```

Comment:
Please don't depend on the value of min/max being 0 in your tests when the result is null. Instead:
```suggestion
  this->AssertMinMaxIsNull(chunked_input2, options);
```

Reply:
done